lldb/test/API/functionalities/data-formatter/setvaluefromcstring/main.m

```diff
@@ -2,10 +2,15 @@
 
 int main() {
     NSDictionary* dic = @{@1 : @2};
+    BOOL b = NO;
     NSLog(@"hello world"); //% dic = self.frame().FindVariable("dic")
     //% dic.SetPreferSyntheticValue(True)
     //% dic.SetPreferDynamicValue(lldb.eDynamicCanRunTarget)
     //% dic.SetValueFromCString("12")
+    //% b = self.frame().FindVariable("b")
+    //% b.SetValueFromCString("YES")
     return 0; //% dic = self.frame().FindVariable("dic")
-    //% self.assertTrue(dic.GetValueAsUnsigned() == 0xC, "failed to read what I wrote")
+    //% self.assertTrue(dic.GetValueAsUnsigned() == 0xC, "failed to read what I
+    //wrote") % b = self.frame().FindVariable("b") %
```

Comment:
Is the formatting here correct? Looks like one line spans two.

Reply:
I think `black` messed that up. Let me fix it and see what the tool says.